Makefile

```diff
@@ -103,13 +103,10 @@ clean: clean-docker clean-py
 clean-docker:
 	docker-compose down -t 60
 	docker system prune -f
-	docker volume prune -f
```

Comment:
This was potentially destructive to peoples other container volumes. Also, we don't use volumes. 😅